Tests/Fuzzers/FuzzTarget.cpp

```diff
@@ -40,7 +40,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
 	if (dumpDataToPcapFile(Data, Size) < 0)
 	{
 		std::cerr << "Can't Dump buffer to a PCAP file!!!!\n";
-		return -1;
+		return 0;
```

Comment:
Why changing the return value to `0`?

Reply:
I misunderstood even if the data is not valid we should add the pattern to the corpus.
see: https://github.com/the-tcpdump-group/libpcap/blob/master/testprogs/fuzz/fuzz_pcap.c